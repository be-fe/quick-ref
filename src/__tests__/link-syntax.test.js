/**
 * @file link-syntax.test
 * @author Cuttle Cong
 * @date 2018/5/21
 * @description
 */
const nps = require('path')

const compile = require('../transpiler/index')

describe('link-syntax', function() {
  it('should link-syntax normal', done => {
    compile(
      nps.join(__dirname, 'fixture/link-syntax.md'),
      {
        linkAlias: {
          avalon: nps.join(__dirname, 'fixture/avalon.md')
        }
      },
      ['link']
    )
      .then(({ output }) => expect(output).toMatchSnapshot())
      .then(done)
  })
})
