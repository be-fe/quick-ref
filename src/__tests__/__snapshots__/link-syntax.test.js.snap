// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`link-syntax should link-syntax normal 1`] = `
"[\`fs\`](avalon.md#ssd)hahahah
@link:avalon说说@hahahah
@link:aval是sswzww@hahahah

\`\`\`markdown
@link:[\`fs\`]avalon#ssd@hahahah
@link:avalon说说@hahahah
@link:aval是sswzww@hahahah
\`\`\`
"
`;
